mediainfo@aclon/files/mediainfo@aclon/mi.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+P=$1
+cd $P
+shift
+F=$(printf '%q ' "$@")
+act=$(dirname $0)
+csv="$act/po/$(echo $LANG | sed 's/.UTF-8//').csv"
+editor="(vim -c 'set filetype=config' -c 'set ignorecase' || less)"
+format="sed 's/ *:/齾:/' | column -t -L -s '齾' -o ''"
+# format="format"                               # Shorter 1st col mode
+cmd="[ -f $csv ] || $act/wget_lang.sh ; mediainfo --Language=file://$csv $F | $format | $editor ; mediainfo $F | $format | $editor"
+# cmd="bash -c \" . $act/func.sh ; $cmd \""     # Shorter 1st col mode
```

Comment:
Actually, `func.sh` is not used here(just for test the render result).It's kept because I thought someone would need it if the first column's blank space is too much. For example:
- with `func.sh`:
```
Name   abc.mp4
Time   01:02:03.004

Average of audio bitrate   128bits/s
Average of video bitrate   4MiB/s
```
- wihout `func.sh`:
```
Name                       abc.mp4
Time                       01:02:03.004

Average of audio bitrate   128bits/s
Average of video bitrate   4MiB/s
```

`func.sh` and all relevant codes should be REMOVED in my next commit.